docs/content/users/providers/lagoon.md

```diff
@@ -0,0 +1,18 @@
+# Amazee Lagoon Integration
+
+DDEV provides integration with the [Amazee Lagoon](https://lagoon.sh/), allowing Lagoon users to quickly download database and files from a Lagoon project to the local DDEV project.
```

Comment:
Being picky, but rewording to avoid “download database” which is uncomfortable phrasing:

```suggestion
DDEV provides integration with the [Amazee Lagoon](https://lagoon.sh/), allowing Lagoon users to quickly download the files and database from a Lagoon project to the local DDEV project.
```

Reply:
You have never liked alliteration :)

Reply:
I probably have but I can’t prove it!